src/components/Common/ResourceCategoryPicker.tsx

```diff
@@ -236,213 +239,238 @@ export function ResourceCategoryPicker({
+        "focus:ring-2 focus:ring-gray-300 focus:ring-offset-2",
+        "transition-all duration-200",
+        disabled && "opacity-50 cursor-not-allowed",
+        className,
+      )}
+      disabled={disabled}
     >
-      <PopoverTrigger asChild>
-        <Button
-          variant="outline"
-          role="combobox"
-          aria-expanded={open}
-          className={cn(
-            "justify-between h-10 min-h-10 px-3 py-2",
-            "hover:bg-gray-50 hover:text-gray-900",
-            "focus:ring-2 focus:ring-gray-300 focus:ring-offset-2",
-            "transition-all duration-200",
-            disabled && "opacity-50 cursor-not-allowed",
-            className,
-          )}
-          disabled={disabled}
-        >
-          <div className="flex items-center gap-2 min-w-0 flex-1">
-            {getDisplayValue()}
-          </div>
-          <ChevronDown
-            className={cn(
-              "h-4 w-4 shrink-0 opacity-50 transition-transform duration-200",
-              open && "rotate-180",
-            )}
-          />
-        </Button>
-      </PopoverTrigger>
+      <div className="flex items-center gap-2 min-w-0 flex-1">
+        {getDisplayValue()}
+      </div>
+      <ChevronDown
+        className={cn(
+          "h-4 w-4 shrink-0 opacity-50 transition-transform duration-200",
+          open && "rotate-180",
+        )}
+      />
+    </Button>
+  );
 
-      <PopoverContent
-        className="w-[420px] p-0 shadow-lg border-0"
-        align="start"
-        sideOffset={4}
-      >
-        <div className="flex flex-col">
-          {/* Header with current location */}
-          <div className="px-4 py-3 border-b bg-gray-50">
-            <div className="flex items-center justify-between">
-              <div className="flex items-center gap-2">
-                <Home className="h-4 w-4 text-gray-500" />
-                <span className="text-sm font-medium text-gray-600">
-                  {getCurrentLevelTitle()}
-                </span>
-                {breadcrumbs.length > 0 && (
-                  <Badge variant="secondary" className="text-xs">
-                    {t("level")} {breadcrumbs.length + 1}
-                  </Badge>
-                )}
-              </div>
-              {value && (
-                <Button
-                  variant="ghost"
-                  size="sm"
-                  onClick={handleClearSelection}
-                  className="h-6 px-2 text-xs text-gray-500 hover:text-gray-700"
-                >
-                  <X className="h-3 w-3 mr-1" />
-                  {t("clear")}
-                </Button>
-              )}
-            </div>
+  const content = (
+    <div className="flex flex-col">
+      {/* Header with current location */}
+      <div className="px-4 py-3 border-b bg-gray-50">
+        <div className="flex items-center justify-between">
+          <div className="flex items-center gap-2">
+            <Home className="h-4 w-4 text-gray-500" />
```

Comment:
```suggestion
            <Home className="size-4 text-gray-500" />
```
change it everywhere

Reply:
https://github.com/ohcnetwork/care_fe/pull/13918#discussion_r2412917671